build.gradle

```diff
@@ -59,6 +63,15 @@ sourceSets {
     }
 }
 
+jar {
+    // OSGi
+    bnd([
+        'Bundle-Vendor': 'CERN',
```

Comment:
Wouldnt it be better to have here 'ossgang'?

Reply:
Updated